package.json

```diff
@@ -57,7 +57,7 @@
     "chai": "^3.5.0",
     "coveralls": "^2.11.8",
     "eslint": "^2.7.0",
-    "eslint-config-airbnb": "^7.0.0",
+    "eslint-config-airbnb": "^8.0.0",
```

Comment:
you may want `eslint-config-airbnb-base` since you don't have the react deps installed